src/components/list/list-renderer.tsx

```diff
@@ -247,7 +247,7 @@ export class ListRenderer {
             <limel-menu
                 class="mdc-deprecated-list-item__meta"
                 items={actions}
-                openDirection="left"
+                openDirection="bottom-end"
```

Comment:
Same thing here.

> Since this can't be set by the consumer, and there is no change to the what the user sees, it's "part of" the main change to limel-portal. So I've squashed this commit into that one.

Reply:
Oh, well, not quite the same here actually, because I added this so that we "keep the same default" here, but I've also kept the separate commit, which changes the default to a new position 😊